recipes/mono-msbuild/all/conanfile.py

```diff
@@ -0,0 +1,93 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, CMake, tools, RunEnvironment
+from conans.errors import ConanInvalidConfiguration
+import glob
+import os
+import yaml
+
+required_conan_version = ">=1.33.0"
+
+
+class MonoConan(ConanFile):
+    name = "mono-msbuild"
+    description = "The Microsoft Build Engine (MSBuild) is the build platform for .NET and Visual Studio."
+    url = "https://github.com/mono/mono"
+    homepage = "https://github.com/mono/msbuild"
+    license = "MIT, BSD"
+    exports_sources = "patches/**"
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "pkg_config"
+    topics = "mono", "dotnet"
+
+    requires = ["mono/6.12.0.122"]
+
+    @property
+    def _minimum_compilers_version(self):
+        # requires C++11
+        return {
+            "gcc": "7",
+            "clang": "6",
+            "apple-clang": "9"
+        }
+
+
+    def validate(self):
+        if self.settings.os == "Windows":
+            raise ConanInvalidConfiguration("mono cannot be built on Windows")
```

Comment:
cannot be built *on* Windows or *for* Windows?

Reply:
It looks like the `mono-msbuild` contains msvc build projects.
So you should probably change this `if` to a check for Visual Studio.